tmt/steps/provision/__init__.py

```diff
@@ -56,21 +56,33 @@
     import tmt.cli
 
 
+def _configure_constant(default: int, envvar: str) -> int:
+    """
+    Deduce the actual value of global constant.
+
+    :param default: the default value of the constant.
+    :param envvar: name of the optional environment variable which would
+        override the default value.
+    :returns: value extracted from the environment variable, or the
+        given default value if the variable did not exist.
+    """
+
+    try:
+        return int(os.environ.get(envvar, default))
+
+    except ValueError as exc:
+        raise tmt.utils.GeneralError(
+            f"Could not parse '{os.environ[envvar]}' as integer.") from exc
+
+
```

Comment:
What about moving this to `utils.py`? It could be useful in general. Seems we already have another place where a similar helper is introduced:

https://github.com/teemtee/tmt/blob/93dc95f3eb88620bc13052adb67c5e8715a0d41b/tmt/utils.py#L5065-L5073

Reply:
Hopefully addressed in 00b8b8db

Reply:
Nice, thanks!